spec/factories/bookable_slots.rb

```diff
@@ -0,0 +1,16 @@
+FactoryGirl.define do
+  factory :bookable_slot do
+    schedule
+    date { Time.zone.now }
+
+    trait :am do
+      start BookableSlot::AM.start
+      add_attribute(:end, BookableSlot::AM.end)
```

Comment:
This is necessary as a work around for the `end` attribute being a reserved word in Ruby.